cookiecutter/ci/{{ cookiecutter.__project_slug }}/.ci/scripts/check_click_for_mypy.py

```diff
@@ -0,0 +1,11 @@
+#!/bin/env python3
+
+import sys
+
+import click
+from packaging.version import parse
+
+if parse(click.__version__) < parse("8.1.1") or parse(click.__version__) >= parse("8.2"):
+    print("🚧 Linting with mypy is currently only supported with click~=8.1.1. 🚧")
```

Comment:
Remind me again what was the mypy issue with `click>=8.2`?

Reply:
I think they changed some annotations in incompatible ways. I found no way to lint with 8.1.1 and 8.2 simultaneously. Maybe it's time to try again.